### ISSUE TITLE
The training and inference results are inconsistent.

### ISSUE THREAD

**Issue by yujian159** [spkgyk/RTFS-Net#4]
Hello,

Thank you very much for your article. I followed your tutorial to download the LRS2 dataset and processed it using preprocess_lrs2.py. I attempted to use the command python train.py --conf-dir config/lrs2_RTFSNet_4_layer.yaml for training, keeping everything at default except not loading the pre-trained weights (frcnn_128_512.backbone.pth.tar).

Upon completing the training, the model has converged, and both the training and validation loss are decreasing. The SI-SDR on the validation set is 12.82 (which is probably due to not using pre-trained weights). However, when I use the trained weights for validation, use python test.py --conf-dir config/lrs2_RTFSNet_4_layer.yaml , the results seem a bit strange:
si-snri mean:-6.394 std:19.8
sdi_i    mean: 4.434  std:8.85
pesq   mean:1.654   std:1.31
stoi     mean:0.5188  std:0.405
si-snr  mean:-6.395  std:19.4
sdr     mean:4.434   std:8.96
testloss   6.3946
The results obtained using train.py and test.py differ too much. Could you please tell me what is causing this? Is such a result normal?
